src/Microsoft.ServiceFabric.Actors/Runtime/ActorStateProviderHelper.cs

```diff
@@ -115,7 +115,7 @@ internal ActorStateProviderHelper(IActorStateProviderInternal owner)
                 }
                 catch (FabricException ex)
                 {
-                    // KVS aborts all active transaction(s) when changing role from primary to secondary
+                    // Aborts all active transaction(s) when changing role from primary to secondary
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.